Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/moth.yml

```diff
@@ -0,0 +1,1210 @@
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 8
+        - ReagentId: Vitamin
+          Quantity: 6
+        - ReagentId: CapsaicinOil
+          Quantity: 1
+#Tastes like cheese, salad, sweet chili
+
+- type: entity
+  name: baked cheese wheel
+  parent: FoodBakingBase
+  id: FoodMothBakedCheese
+  description: A baked cheese wheel, melty and delicious..
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cheesy
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 18
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Vitamin
+          Quantity: 4
+        - ReagentId: Protein
+          Quantity: 1
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: bakedcheese
+
+- type: entity
+  name: baked cheese platter
+  parent: FoodBakingBase
+  id: FoodMothBakedCheesePlatter
+  description: A favourite for sharing. Usually served with crispy bread slices for dipping, because the only thing better than good cheese is good cheese on bread.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cheesy
+      - bread
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: bakedcheeseplatter
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 25
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 16
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: Protein
+          Quantity: 2
+
+- type: entity
+  name: big baked rice
+  parent: FoodBakingBase
+  id: FoodMothBakedRice
+  description: A mothic favourite, baked rice can be filled with a variety of vegetable fillings to make a delicious meal to share.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - potato
+      - cabbage
+      - rice
+  - type: SliceableFood
+    count: 6
+    slice: FoodMothBakedRicePortion
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: bakedrice
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 12
+        - ReagentId: Protein
+          Quantity: 6
+        - ReagentId: Vitamin
+          Quantity: 12
+
+- type: entity
+  name: lil baked rice
+  parent: FoodBakingBase
+  id: FoodMothBakedRicePortion
+  description: A single portion of baked rice, perfect as a side dish, or even as a full meal.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - potato
+      - cabbage
+      - rice
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: bakedrice-portion
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 5
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 2
+        - ReagentId: Protein
+          Quantity: 1
+        - ReagentId: Vitamin
+          Quantity: 2
```

Comment:
0.7

Reply:
changed it to 1.25 because 8 divided in 6 'slice' will result in 1.33 vitamin per slice